### PR TITLE
cerf instead of Faddeeva

### PR DESCRIPTION
**Pull request by loriab**
This is roughly what I was thinking of for #53 . See what you think. (PR includes #52 but I can rebase those out later.)

### CODE REVIEW

**Comment by robashaw** [comment]
All good, thanks for the changes.

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -53,10 +53,29 @@ target_include_directories(ecpint
     $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include>
 )
 
-target_link_libraries(ecpint
- PRIVATE
-  Faddeeva
-)
+if (LIBECPINT_USE_CERF)
+    find_package(cerf CONFIG REQUIRED) # QUIET)  # COMPONENTS C)
```
Comment (junghans) [inline src/CMakeLists.txt:57]: @ loriab Which version of libcerf comes with a `cerfConfig.cmake`?

Comment (junghans) [inline src/CMakeLists.txt:57]: @loriab  @jwuttke Which version of libcerf comes with a `cerfConfig.cmake`?

Comment (junghans) [reply src/CMakeLists.txt:57]: Got it: https://jugit.fz-juelich.de/mlz/libcerf/-/merge_requests/2

Comment (junghans) [reply src/CMakeLists.txt:57]: I patched the above PR into the Fedora and openSUSE RPM, so we could bump libecpint.

**Pull request merged**